src/python/pants/build_graph/target.py

```diff
@@ -836,18 +835,6 @@ def closure(self, *vargs, **kwargs):
     """
     return self.closure_for_targets([self], *vargs, **kwargs)
 
-  # TODO(Eric Ayers) As of 2/5/2015 this call is DEPRECATED and should be removed soon
-  def add_labels(self, *label):
```

Comment:
Would it be ok to leave this method here as a noop with a deprecation warning? I suspect that because it was deprecated pre-deprecation warnings, some folks might not have gotten the message.

Reply:
OTOH, for the same reason, these where never public API, so I don't think it's required. If we start chasing down "methods that people may have thought were public API before we explicitly marked public methods as such" there's no end to it...